docs/source/release_notes.rst

```diff
@@ -11,6 +11,7 @@ Release Notes
         * Rename ``ZIPCode`` logical type to ``PostalCode`` (:pr:`741`)
         * Fix issue with smart-open version 5.0.0 (:pr:`750`, :pr:`758`)
         * Update minimum scikit-learn version to 0.22 (:pr:`763`)
+        * Drop support for Python version 3.6 (:pr:`768`)
```

Comment:
Let's add something like this below Future Release. We did this [for Featuretools](https://github.com/alteryx/featuretools/blob/main/docs/source/release_notes.rst)
```
    .. warning::
        The next non-bugfix release of Featuretools will not support Python 3.6
```

Reply:
Does that mean that dropping 3.6 support shouldn't go into the upcoming release, then?

Reply:
Sorry, I mean we should put something like this
```
    .. warning::
        This release Woodwork does not support Python 3.6
```